### PR TITLE
stty: add page

### PR DESCRIPTION
**Pull request by RafRaf**
For #2213 

----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by MasterOdin** [comment]
Most every time I've ever had to lookup and use this command, it was because my terminal had gotten messed up and I needed to reset it, which was done by using the `stty sane` command, which might be worth mentioning here?

**Comment by RafRaf** [comment]
@sbrl, thanks for your feedback. Maybe better to add a `sane` instead of `echo` command?

@MasterOdin, thanks for your suggestion.

PS: PR updated

File: pages/common/stty.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`stty rows {{rows}}`
+
+- Set the number of columns:
+
+`stty cols {{cols}}`
+
+- Get the only actual transfer speed of a device:
+
+`stty -f /dev/stdout speed`
```
Comment (agnivade) [inline pages/common/stty.md]: /dev/stdout in tokens (`{{}}`)

File: pages/common/stty.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Set the number of rows:
+
+`stty rows {{rows}}`
+
+- Set the number of columns:
+
+`stty cols {{cols}}`
+
+- Get the only actual transfer speed of a device:
```
Comment (sbrl) [inline pages/common/stty.md]: Ah, this is much clearer - thanks! Perhaps it would read better as `Get the actual`.....?

**Comment by RafRaf** [comment]
Sorry for the delay. I've updated the PR.
@agnivade , I'm not sure that stty available for Windows. Do I need to move stty.md to Linux, SunOS, and MacOS folders?

**Comment by agnivade** [comment]
If it's not available, then yes.

**Comment by sbrl** [comment]
Thanks, @RafRaf :smiley_cat:

**Pull request merged**